File: TRIANGLE PATTERN 1/code9.c
#include <stdio.h>

void main (){

	int row ,temp = 1,temp1=1;
	printf("Enter your no. of rows : ");
	scanf("%d",&row);

	for(int i = 1; i <= row ; i++){

		
		for(int j = 1 ; j < i + 1;j++){
			temp =1;
			for(int a = 1; a <= j ;a++){

				temp *= a; 
			}

			printf("%d\t",(i)*temp);
					
		}
		printf("\n\n");
	}
}

File: JAVA DAILYCODES ASSGNPRATICE QUIZE DSA/Basic is else assignments/8/code9.java
import java.util.Scanner;

class NestedDemo{

	public static void main(String[] args){

		Scanner s = new Scanner(System.in);
		System.out.println("enter no. of rows");
		int row = s.nextInt();
		int temp = 1;


		for(int i = 0; i < row ; i++){
			for(int j = 0;j <row ;j++){
                                                                  
				if(i % 2==0){

					System.out.print((j+1)+((char)(65+j)+"		"));

				}else{
					System.out.print((row-j)+((char)(64+row-j)+"		"));
				}

				temp++;
			}
			
			
			System.out.println("\n");
		}
	}
}
